components/Downloads/ChangelogModal/index.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import { VFile } from 'vfile';
+
+import Button from '@/components/Common/Button';
+import ChangelogModal from '@/components/Downloads/ChangelogModal';
+import { MDXRenderer } from '@/components/mdxRenderer';
+import { compileMDX } from '@/next.mdx.compiler.mjs';
+import { githubProfileAvatarUrl } from '@/util/gitHubUtils';
+
+type Story = StoryObj<typeof ChangelogModal>;
+type Meta = MetaObj<typeof ChangelogModal>;
+
+const names = [
+  'ovflowd',
+  'bmuenzenmeyer',
+  'AugustinMauroy',
+  'HinataKah0',
+  'Harkunwar',
+  'rodion-arr',
+  'mikeesto',
+  'bnb',
+  'benhalverson',
+  'aymen94',
+  'shanpriyan',
+  'Wai-Dung',
+  'manishprivet',
+  'araujogui',
+  'thefossildev',
+];
+
+const children = `
```

Comment:
With this I just realised the following:

We don't have styles for headings and paragraphs and etc.

We need to add h1, h2, h3, h4, h5, h6, p (margins) styles.

The `<pre>` styles we shouldn't worry, as they will come with the codebox component PR

Reply:
(Feel free to add the needed styles in this PR)